src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BQSRIntegrationTest.java

```diff
@@ -0,0 +1,60 @@
+package org.broadinstitute.hellbender.tools.walkers.bqsr;
+
+import htsjdk.samtools.ValidationStringency;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.Main;
+import org.broadinstitute.hellbender.tools.validation.CompareBaseQualities;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+//Note: has to be in this package to have access to Main.instanceMain
+public final class BQSRIntegrationTest extends CommandLineProgramTest {
```

Comment:
Yes, this one tests both steps

On Wed, Apr 20, 2016 at 9:56 PM, droazen <notifications@github.com> wrote:

> In
> src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BQSRIntegrationTest.java
> <https://github.com/broadinstitute/gatk/pull/1745#discussion_r60516678>:
>
> > @@ -0,0 +1,60 @@
> > +package org.broadinstitute.hellbender.tools.walkers.bqsr;
> > +
> > +import htsjdk.samtools.ValidationStringency;
> > +import org.broadinstitute.hellbender.CommandLineProgramTest;
> > +import org.broadinstitute.hellbender.Main;
> > +import org.broadinstitute.hellbender.tools.validation.CompareBaseQualities;
> > +import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
> > +import org.broadinstitute.hellbender.utils.test.BaseTest;
> > +import org.testng.Assert;
> > +import org.testng.annotations.Test;
> > +
> > +import java.io.File;
> > +
> > +//Note: has to be in this package to have access to Main.instanceMain
> > +public final class BQSRIntegrationTest extends CommandLineProgramTest {
>
> We already have a BaseRecalibratorIntegrationTest, and you've now created
> BQSRIntegrationTest??
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/broadinstitute/gatk/pull/1745/files/1e699bafbb8d12bd3d46a5fe91acf710051ef940#r60516678>
>